src/settings-compat-v7/_color.scss

```diff
@@ -0,0 +1,11 @@
+// --- Colors --- //
```

Comment:
Please comment any weird thing you see. These colors don't seem to be used by components.
$c-highlighted is apparently not used anywhere
$c-featured is used in mt for as category tags, but Julian said the categories should be in each project, not in the global them